### DISCUSSION TITLE
Add support for UpgradeableHoppers

### DISCUSSION THREAD

**Discussion by PSYCHEER** [GriefPrevention/GriefPrevention#2298] [Ideas]
Hi, I have one plugin which uses land claim plugin (Lands) as integration to give trusted members ability to edit hopper trough GUI.
Lame is that plugin have lack of support for GP and when members are trusted, they don't receive permission to edit hopper/upgrade or look into.
I've asked in support to get answer if he will add this support as it's premium plugin and it's not cheap.
But no, bcz he have his own land claim plugin etc.

So my question is, if it can be for GP possible to add support for this plugin?

![image](https://github.com/GriefPrevention/GriefPrevention/assets/42911029/a2dd1c3a-48c2-4cc4-9a2e-29aa85a18d4b)

**Comment by Jikoo** [comment]
An addon could do this. You may need to hire someone to do that though, I doubt anyone working on free open source software is going to be excited about writing a compatibility addon for someone else's paid closed source plugin.

**Reply by PSYCHEER** [reply]
Thats what I wanted to know. Thanks for your time and I dont go to this paid plugin then...
